Bluetooth: mesh: remove relative paths in samples (#11796)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

  * Fixed an issue with the sample not fitting into RAM size on the nrf52dk_nrf52832 board.

>   * Fixed an issue where samples copied via the VS Code extension did not compile due to relative paths in CMakeLists, referencing files outside the applications folder.

Reviewer: This should be pulled in with 2 spaces, as it does not have anything to to with sensor_client.

> * Fixed an issue where some samples copied via the VS Code extension did not compile due to relative paths in CMakeLists, referencing files outside the applications folder.

Reviewer 2: ```suggestion
* Fixed an issue where some samples copied using the `nRF Connect for Visual Studio Code`_ extension would not compile due to relative paths in :file:`CMakeLists.txt`, which were referencing files outside of the applications folder.
```
